src/main/java/org/adoptopenjdk/maven/plugins/JsplitpkgscanMojo.java

```diff
@@ -55,35 +59,39 @@ public void execute() throws MojoExecutionException {
     }
 
     void runJsplitpkgscan(Tool tool) {
-        Set<String> checkedScopes = Set.of("compile", "runtime");
-        getLog().debug("project: " + project);
-        getLog().debug("localRepository: " + localRepository);
-        getLog().debug("outputDirectory: " + outputDirectory);
-        getLog().debug("artifact: " + project.getArtifact());
-        getLog().debug("artifact file: " + project.getArtifact().getFile());
+        Log log = getLog();
+        if (scopes == null || scopes.isEmpty()) {
+            log.debug("Using default scopes");
```

Comment:
might as well say what those default scopes are :-)

Reply:
@karianna see line 70, there we log the set scopes as debug